### ISSUE TITLE
EXR Layers / AOV not found

### ISSUE THREAD

**Issue by luisfos** [darbyjohnston/DJV#501]
DJV isn't showing the layers/aov of EXRs, we can see them correctly in other software like houdini / nuke.

**Comment by tiltX** [comment]
Can't test this right now, but is the EXR file by any chance a multi-part file? Those are not supported by djv it seems.

**Comment by luisfos** [comment]
yes its multi-part, is it possible to make it work?On 10 Feb 2024 14:34, tiltX ***@***.***> wrote:
Can't test this right now, but is the EXR file by any chance a multi-part file? Those are not supported by djv it seems.

**Comment by darbyjohnston** [comment]
HI, and sorry for the long delay; I just released a new version, 3.0.0, which has support for multi-part EXR files. I'm going to close this as completed, but feel free to create new issues if you find any problems with the new version.
